processing file asmFiles/beq1.asm (1 of 29)
processing file asmFiles/beq2.asm (2 of 29)
processing file asmFiles/bge1.asm (3 of 29)
processing file asmFiles/bge2.asm (4 of 29)
processing file asmFiles/bgeu1.asm (5 of 29)
processing file asmFiles/bgeu2.asm (6 of 29)
processing file asmFiles/blt1.asm (7 of 29)
processing file asmFiles/blt2.asm (8 of 29)
processing file asmFiles/bltu1.asm (9 of 29)
processing file asmFiles/bltu2.asm (10 of 29)
processing file asmFiles/bne1.asm (11 of 29)
processing file asmFiles/bne2.asm (12 of 29)
processing file asmFiles/cfHazard.asm (13 of 29)
processing file asmFiles/count_days.asm (14 of 29)
--- memsim.hex	2025-10-08 14:23:56.421795774 -0400
+++ memcpu.hex	2025-10-08 14:23:56.415795542 -0400
@@ -1,23 +1,22 @@
-:0400000001700293F6
-:040001000080031365
-:040002007E90039356
-:04000300FFF30313F1
-:040004007D000E93DA
-:0400050041D383B3AD
-:0400060001E00F13F3
-:04000700FFC1011321
-:0400080001E12023CF
-:04000900FFC101131F
-:04000A00006120234E
-:04000B00038000EF7F
-:04000C0000012F833D
-:04000D00004101139A
-:04000E0016D00513F0
-:04000F00FFC1011319
-:0400100000A1202308
-:04001100FFC1011317
-:040012000071202336
-:04001300018000EF79
+:0400010000000020DB
+:04000200000000609A
+:04000300000000C039
+:0400040000000140B7
+:04000500000001E016
+:04000600000002A054
+:040007000000038072
+:040008000000048070
+:04000900000005A04E
+:04000A0000000019D9
+:04000B0000000840A9
+:04000C00000009C027
+:04000D0000000B6084
+:04000E0000000D20C1
+:04000F0000000F00DE
+:0400100000001100DB
+:0400110000001320B8
+:040012000000156075
+:04001300000017C012
 :04001400000125833F
 :040015000041011392
 :0400160001F286333A
@@ -38,6 +37,5 @@
 :04002500FFC1011303
 :0400260000F12023A2
 :0400270000008067EE
-:043FFE0000000019A6
-:043FFF00000023A5F6
+:043FFE0000000007B8
 :00000001FF
processing file asmFiles/dataHazard.asm (15 of 29)
--- memsim.hex	2025-10-08 14:23:58.578878868 -0400
+++ memcpu.hex	2025-10-08 14:23:58.572878637 -0400
@@ -13,6 +13,6 @@
 :04000C00000E04637B
 :04000D0001DE8EB3CF
 :04000E00FFFFFFFFF2
-:0403C8000000000928
+:0403C800000000032E
 :0403C9000000000927
 :00000001FF
processing file asmFiles/fib.asm (16 of 29)
--- memsim.hex	2025-10-08 14:24:00.549954797 -0400
+++ memcpu.hex	2025-10-08 14:24:00.541954489 -0400
@@ -14,27 +14,5 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/halt.asm (17 of 29)
processing file asmFiles/jal.asm (18 of 29)
processing file asmFiles/jalr.asm (19 of 29)
